server/src/handlers/http/middleware.rs

```diff
@@ -164,6 +171,31 @@ where
         /* ## Section end */
 
         let auth_result: Result<_, Error> = (self.auth_method)(&mut req, self.action);
+        let body = json!([
+            {
+                "version": "1.0",
+                "user-agent":&req
+                .headers()
+                .get("user-agent")
+                .and_then(|value| value.to_str().ok())
+                .unwrap_or("unknown"),
+                "datetime": Utc::now(),
+                "action":self.action,
+                "Actor":{
+                    "type": &req
+                    .headers()
+                    .get("user-agent")
+                    .and_then(|value| value.to_str().ok())
+                    .unwrap_or("unknown"),
+                    "id": "user123"
```

Comment:
Why is this hardcoded, and what is the intention behind using user-agent as type?